lib/kitchen/driver/openstack.rb

```diff
@@ -325,11 +323,15 @@ def parse_ips(pub, priv)
 
       def add_ohai_hint(state)
         info 'Adding OpenStack hint for ohai'
-        ssh = Fog::SSH.new(*build_ssh_args(state))
-        ssh.run([
-          %(sudo mkdir -p #{Ohai::Config[:hints_path][0]}),
-          %(sudo touch #{Ohai::Config[:hints_path][0]}/openstack.json)
-        ])
+        mkdir_cmd = "sudo mkdir -p #{hints_path}"
+        touch_cmd = "sudo touch #{hints_path}/openstack.json"
+        instance.transport.connection(state).execute(
```

Comment:
You could offer windows equivalents of these commands.  mkdir could just be ```"mkdir #{hints_path}"``` and touch could be ```" '' > #{hints_path}\\openstack.json"```  (just off the top of my head.